src/common/status.h

```diff
@@ -168,7 +168,7 @@ struct StringInStatusOr<T, std::enable_if_t<sizeof(T) < sizeof(std::string)>> :
   StringInStatusOr(StringInStatusOr<U>&& v) : BaseType(new std::string(*std::move(v))) {}  // NOLINT
   template <typename U, typename std::enable_if_t<!StringInStatusOr<U>::inplace, int> = 0>
   StringInStatusOr(StringInStatusOr<U>&& v)  // NOLINT
-      : BaseType((typename StringInStatusOr<U>::BaseType &&)(std::move(v))) {}
+      : BaseType((typename StringInStatusOr<U>::BaseType&&)(std::move(v))) {}
```

Comment:
Could you review this change?

Reply:
It's the result of `./x.py format`

Reply:
@wsehjk The format is introduced by clang-format-16 and Kvrocks is now using clang-format-14, so you can change the clang-format to 14 for making the CI happy.

Reply:
Ok, i will reinstall clang-format-14